target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-ix4-200d.dts

```diff
@@ -0,0 +1,204 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "kirkwood.dtsi"
+#include "kirkwood-6281.dtsi"
+
+/ {
+	model = "Iomega ix4-200d";
+	compatible = "iom,ix4-200d", "marvell,rd88f6281-a", "marvell,rd88f6281", "marvell,kirkwood-88f6281", "marvell,kirkwood";
+
+memory {
+	device_type = "memory";
+	reg = <0x00000000 0x20000000>;
+};
+
+chosen {
+	bootargs = "console=ttyS0,115200n8 earlyprintk";
+	stdout-path = &uart0;
+};
+
+mbus {
+	pcie-controller {
```

Comment:
Please use aliases where is possible.

Reply:
Please look here: https://github.com/openwrt/openwrt/blob/a872570c043f5beee21c3af695f1a5a77f7eab04/target/linux/kirkwood/files/arch/arm/boot/dts/kirkwood-nsa310s.dts#L260

You can do it simpler:
```
&pciec {
	status = "okay";
};

&pcie0 {
	status = "okay";
};
```